Python/bytecodes.c

```diff
@@ -4778,6 +4787,26 @@ dummy_func(
             DEOPT_IF(func->func_version != func_version);
         }
 
+        tier2 op(_LOAD_GLOBAL_MODULE, (index/1 -- res, null if (oparg & 1))) {
+            PyDictObject *dict = (PyDictObject *)GLOBALS();
+            PyDictUnicodeEntry *entries = DK_UNICODE_ENTRIES(dict->ma_keys);
+            PyObject *res_o = entries[index].me_value;
+            DEOPT_IF(res_o == NULL);
+            Py_INCREF(res_o);
+            res = PyStackRef_FromPyObjectSteal(res_o);
```

Comment:
We'll need to also update this pattern to make it thread-safe, but that can happen in subsequent PRs.

Reply:
Yep, I'm trying to keep PRs small and isolated to help minimize reviewer load.